### PR TITLE
Issue 13295 set sidebar width in edit layout page

### PR DESCRIPTION
**Pull request by luisrojascr**

### CODE REVIEW

File: src/app/portlets/dot-edit-page/layout/dot-edit-layout/dot-edit-layout.component.scss [hunk 1]
```diff
@@ -87,9 +87,31 @@
 }
 
 [class^="dot-edit-layout__template-sidebar"] {
+    justify-content: flex-end;
+
+    span {
```
Comment (fmontes) [inline src/app/portlets/dot-edit-page/layout/dot-edit-layout/dot-edit-layout.component.scss]: You sure you need this?

File: src/app/portlets/dot-edit-page/layout/dot-edit-layout/dot-edit-layout.component.ts [hunk 2]
```diff
@@ -171,6 +171,7 @@ export class DotEditLayoutComponent implements OnInit {
         this.isModelUpdated = false;
         this.form.valueChanges.subscribe(() => {
             this.isModelUpdated = !_.isEqual(this.form.value, this.initialFormValue.value);
+            // TODO: Set sidebar to null if sidebar location is empty
```
Comment (fmontes) [inline src/app/portlets/dot-edit-page/layout/dot-edit-layout/dot-edit-layout.component.ts]: Add more explaining here, something like "we're expecting a change in the backend bla bla bla"

File: src/app/portlets/dot-edit-page/layout/dot-sidebar-properties/dot-sidebar-properties.component.html [hunk 3]
```diff
@@ -0,0 +1,11 @@
+<p-overlayPanel #op>
+    <div>
```
Comment (fmontes) [inline src/app/portlets/dot-edit-page/layout/dot-sidebar-properties/dot-sidebar-properties.component.html]: Why do you need double <div>?

File: src/app/portlets/dot-edit-page/layout/dot-sidebar-properties/dot-sidebar-properties.component.html [hunk 4]
```diff
@@ -0,0 +1,11 @@
...
+        <div class="dot-sidebar-properties__radio-buttons-container" [formGroup]="group">
+            <p-radioButton name="sidebarWidth" value="small" [label]="dotMessageService.get('editpage.layout.sidebar.width.small')" formControlName="width"></p-radioButton>
+            <p-radioButton name="sidebarWidth" value="medium" [label]="dotMessageService.get('editpage.layout.sidebar.width.medium')" formControlName="width"></p-radioButton>
+            <p-radioButton name="sidebarWidth" value="large" [label]="dotMessageService.get('editpage.layout.sidebar.width.large')" formControlName="width"></p-radioButton>
+        </div>
+    </div>
+</p-overlayPanel>
+
+<button pButton type="button" icon="fa-ellipsis-v" (click)="op.toggle($event)"></button>
```
Comment (fmontes) [inline src/app/portlets/dot-edit-page/layout/dot-sidebar-properties/dot-sidebar-properties.component.html]: we need to add some text to the button icons to make it accessibles.

File: src/app/portlets/dot-edit-page/layout/dot-sidebar-properties/dot-sidebar-properties.component.ts [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+    constructor(private dotMessageService: DotMessageService) {}
+
+    ngOnInit() {
+        this.dotMessageService.getMessages([
+            'editpage.layout.sidebar.width.small',
+            'editpage.layout.sidebar.width.medium',
+            'editpage.layout.sidebar.width.large'
+        ]).subscribe();
+    }
+}
```
Comment (freddyucv) [inline src/app/portlets/dot-edit-page/layout/dot-sidebar-properties/dot-sidebar-properties.component.ts]: should not this component implement ControlValueAccedor?

File: src/app/portlets/dot-edit-page/layout/dot-sidebar-properties/dot-sidebar-properties.component.spec.ts [hunk 6]
```diff
@@ -0,0 +1,46 @@
...
+        const pOverlayPanel = fixture.debugElement.query(By.css('p-overlaypanel'));
+
+        expect(pOverlayPanel).toBeDefined();
+    });
+
+    it('should has 3 radio buttons', () => {
+        const radioButtons = fixture.debugElement.query(By.css('.dot-sidebar-properties__radio-buttons-container'));
+
+        expect(radioButtons.children.length).toEqual(3);
+    });
```
Comment (freddyucv) [inline src/app/portlets/dot-edit-page/layout/dot-sidebar-properties/dot-sidebar-properties.component.spec.ts]: we have to test that the tree radio buttons have the right values, we have to test the button's click too

**Pull request merged**